### ISSUE TITLE
UI/UX components

### ISSUE THREAD

**Issue by crzypatchwork** [hicetnunc2000/hicetnunc#757]
- [x] </> icon on interactive objkts
- [ ] menu/header changes
- [ ] play icon for videos/3d icons review
- [x] galleries fix
- [x] pagination for creations/collections
- [x] loading centralized horizontally/vertically :)) ( +/- )
- [x] different tags/sizes for texts depending on title/paragraph/warnings ( +/- )
- [x] css on settings/config/explore page
- [ ] profiles royalties/sales activity
**objkts**
- [ ] history along tx hash
- [x] Height limit https://github.com/hicetnunc2000/hicetnunc/blob/main/src/pages/objkt-display/index.js ( +/- )
- [ ] audio player (waves + images) https://www.npmjs.com/package/@dschoon/react-waves
**settings**
- [ ] description of components and their functions 
**minting**
- [ ] unlockable content
- [ ] minting with multiple stages (each one minimal) for uploading display Uris (for music (audio + image), for unlockables (3d + usdz + video + image))
- [x] description inputs with flexible size
- [ ] Sensitive contents tags+filter/"NFT license"/terms of service
**accessibility**
- [ ] error handling
- [ ] more information on FAQ/about
- [ ] info tooltips
- [x] search/explore
- [ ] hicetnunc context -> sdk

**Comment by andrevenancio** [comment]
Can we move the `/tags/SOMETAG` page to `/explore/SOMETAG` ? I think a explore page would be nice

**Comment by crzypatchwork** [comment]
yes we could actually use only /explore instead of having /search /tags

**Comment by violetforest** [comment]
https://github.com/hicetnunc2000/hicetnunc/pull/751 heres some HTML tags for accessibility

**Comment by violetforest** [comment]
" fetch more information on FAQ/about"

what does this mean @crzypatchwork ?

**Comment by crzypatchwork** [comment]
I would suggest us bringing informations about what is the project, informations about fees, about the smart contract addresses, as well some tools for devs/getting metadata

**Comment by sf-pixel** [comment]
Working on FAQ rn, can add popups in a seperate PR later. Will be active about this in #dev in the Discord.

**Comment by sf-pixel** [comment]
> Working on FAQ rn, can add popups in a seperate PR later. Will be active about this in #dev in the Discord.

FAQ abandoned. Popups going stale...

**Comment by crzypatchwork** [comment]
why FAQ abandoned?

**Comment by sf-pixel** [comment]
After the first two points I did on the FAQ, I realized that I'm not a good FAQ writer... we should probably get a copywriter or writer to write something up so that it sounds a bit better. Anywho, I've finished the new menu from hicathon, if you want to merge it it's over [here](https://github.com/hicetnunc2000/hicetnunc/pull/1007)

**Comment by crzypatchwork** [comment]
haven't you write the hicathon things though? about the menu let's see how to adapt the design after the explore function being enabled ty

**Comment by sf-pixel** [comment]
> haven't you writen the hicathon.xyz things though? about the menu let's see how to adapt the design after the explore function being enabled ty

Yep, I did write it but it seems I'm better with guidelines than I am with FAQ since I'm good at telling what to do/not do instead of how to do it. I could try again... may not be the same as the hicathon guidelines tho

**Comment by scalemodal** [comment]
> about the menu let's see how to adapt the design after the explore function being enabled ty

Do you want to implement the explore function by yourself or it's an open task to grab by anyone else still?
For me to understand what are the next steps and how it may affect the menu or header, seems like by only adding a search field.

**Comment by crzypatchwork** [comment]
sorry for the late response @denscimonk. the search feature is being implemented here https://github.com/hicetnunc2000/hicetnunc/tree/main/src/pages/search any suggestions are welcome

**Comment by scalemodal** [comment]
@crzypatchwork Hi, I'm checking the explore function, these tags are the suggestions based on keywords you are trying to search? For now they are static seems: https://www.hicetnunc.xyz/search

![image](https://user-images.githubusercontent.com/81647978/129600037-93992a5c-0608-481a-97c2-74e57f7f87f9.png)
